client/app/scripts/charts/nodes-chart.js

```diff
@@ -391,16 +390,19 @@ class NodesChart extends React.Component {
   }
 }
 
+
 function mapStateToProps(state) {
   return {
-    adjacentNodes: getAdjacentNodes(state),
+    nodes: nodeAdjacenciesSelector(state),
```

Comment:
Strange! Adjacencies are not changing? (`update`). Will try and create a test case kind of thing...